### PR TITLE
New Civ Faction

### PR DESCRIPTION
**Pull request by MatrikSky**

### CODE REVIEW

File: Vindicta.Altis/Templates/Factions/WW2_CIVILIAN.sqf [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+_veh set [T_VEH_SIZE-1, nil];
+_veh set [T_VEH_default, [
+    "LIB_GazM1",
+    "LIB_GazM1_dirty",
+    "LIB_FRA_CitC4",
+    "LIB_FRA_CitC4Ferme"
+]];
+_vehWeights = [];
+_vehWeights set [T_VEH_SIZE-1, nil];
+_vehWeights set [T_VEH_default, [
```
Comment (billw2012) [inline Vindicta.Altis/Templates/Factions/WW2_CIVILIAN.sqf]: You need to update this bit as well, it represents the chance of spawning for each type. It should be same length array as T_VEH_default above.

File: Vindicta.Altis/Templates/initFactions.sqf [hunk 2]
```diff
@@ -19,8 +19,10 @@
 ["Templates\Factions\WW2_Heer.sqf", T_FACTION_Military]				call t_fnc_initializeTemplateFromFile;
 ["Templates\Factions\WW2_UK.sqf", T_FACTION_Military]				call t_fnc_initializeTemplateFromFile;
 
+
 // Other factions
-["Templates\Factions\CIVILIAN.sqf", T_FACTION_Civ]					call t_fnc_initializeTemplateFromFile;
+["Templates\Factions\WW2_CIVILIAN.sqf", T_FACTION_Civ]					call t_fnc_initializeTemplateFromFile;
+"Templates\Factions\CIVILIAN.sqf", T_FACTION_Civ]					call t_fnc_initializeTemplateFromFile;
```
Comment (billw2012) [inline Vindicta.Altis/Templates/initFactions.sqf]: Missing square bracket here.

File: Vindicta.Altis/Templates/initFactions.sqf [hunk 3]
```diff
@@ -19,8 +19,10 @@
 ["Templates\Factions\WW2_Heer.sqf", T_FACTION_Military]				call t_fnc_initializeTemplateFromFile;
 ["Templates\Factions\WW2_UK.sqf", T_FACTION_Military]				call t_fnc_initializeTemplateFromFile;
 
+
 // Other factions
-["Templates\Factions\CIVILIAN.sqf", T_FACTION_Civ]					call t_fnc_initializeTemplateFromFile;
+["Templates\Factions\WW2_CIVILIAN.sqf", T_FACTION_Civ]					call t_fnc_initializeTemplateFromFile;
```
Comment (billw2012) [inline Vindicta.Altis/Templates/initFactions.sqf]: Indentation messed up here.

File: Vindicta.Altis/Templates/initFactions.sqf [hunk 4]
```diff
@@ -29,4 +31,4 @@
 ["Templates\Factions\RHS_LDF_ranger.sqf", T_FACTION_Police]			call t_fnc_initializeTemplateFromFile;
 ["Templates\Factions\RHS_LDF_NATO_police.sqf", T_FACTION_Police] 	call t_fnc_initializeTemplateFromFile;
 ["Templates\Factions\WW2_Heer_police.sqf", T_FACTION_Police]		call t_fnc_initializeTemplateFromFile;
-["Templates\Factions\WW2_UK_police.sqf", T_FACTION_Police]			call t_fnc_initializeTemplateFromFile;
```
Comment (billw2012) [inline Vindicta.Altis/Templates/initFactions.sqf]: Looks like end of file newline was removed here? Perhaps by accident but it is good practice to end file with newline.

File: Vindicta.Altis/Templates/Factions/WW2_CIVILIAN.sqf (hunk 1 above)
Comment (MatrikSky) [reply Vindicta.Altis/Templates/Factions/WW2_CIVILIAN.sqf]: on it

File: Vindicta.Altis/Templates/initFactions.sqf (hunk 2 above)
Comment (MatrikSky) [reply Vindicta.Altis/Templates/initFactions.sqf]: done

File: Vindicta.Altis/Templates/Factions/WW2_CIVILIAN.sqf (hunk 1 above)
Comment (MatrikSky) [reply Vindicta.Altis/Templates/Factions/WW2_CIVILIAN.sqf]: done

File: Vindicta.Altis/Templates/initFactions.sqf [hunk 5]
```diff
@@ -6,27 +6,28 @@
 // !! Factions will be listed in UI in the same order as here !! 
 
 // Military factions
```
Comment (billw2012) [inline Vindicta.Altis/Templates/initFactions.sqf]: All this indentation is messed up, are you using 2 space tabs instead of 4?

Comment (MatrikSky) [reply Vindicta.Altis/Templates/initFactions.sqf]: I'm using 4 space tabs

**Comment by billw2012** [comment]
Accepting it, but there is no UI for selecting civs yet!

**Pull request merged**